src/index.html

```diff
@@ -1,13 +1,427 @@
+
+      <section class="page__section contact" id="Contacts">
+        <div class="container">
+          <h2 class="page__section-title page__section-title--margin">Contact us</h2>
+
+          <div class="grid grid--desktop grid--pad">
+            <div class="grid__item grid__item-desktop--1-3 grid__item-pad--1-3">  
+              <div class="contact__info">
+                <div class="contact__address">
+                  The Met Fifth Avenue
+                </div>
+                <div class="contact__info-title">
+                  Phone
+                </div>
+                <a 
+                  href="tel:+1 212-535-7710" 
+                  class="contact__info-link"
+                >
+                  +1 212-535-7710
+                </a>
+              </div>
+
+              <div class="contact__info">
+                <div class="contact__info-title">
+                  Address
+                </div>
+                
+                <div class="contact__info-link">
+                  1000 Fifth Avenue 
+                  <br> New York, NY 10028
+                </div>
+              </div>
+            </div>
+            <div class="grid__item grid__item-desktop--4-6 grid__item-pad--1-3">
+              <div class="contact__info">
+                <div class="contact__address">
+                  The Met Cloisters
+                </div>
+                <div class="contact__info-title">
+                  Phone
+                </div>
+                <a 
+                  href="tel:+1 212-923-3700" 
+                  class="contact__info-link"
+                >
+                  +1 212-923-3700
+                </a>
+              </div>
+
+              <div class="contact__info contact--last">
+                <div class="contact__info-title">
+                  Address
+                </div>
+                
+                <div class="contact__info-link">
+                  99 Margaret Corbin Drive 
+                  <br>
+                  Fort Tryon Park 
+                  <br>
+                  New York, NY 10040
+                </div>
+              </div>
+            </div>
+            <div class="grid__item grid__item-desktop--8-12 grid__item-pad--4-6">
+              <form action="#">
+                <input 
+                  name="Name"
+                  placeholder="Name" 
+                  type="text"
+                  class="contact-field form-field"
+                >
+                <input 
+                  name="Email" 
+                  placeholder="Email"
+                  type="email"
+                  class="contact-field form-field"
+                >
+                <textarea 
+                  name="Message"
+                  placeholder="Message"
+                  class="
+                    contact-field
+                    contact-field--last
+                    form-field 
+                    form-field--text"
+                >
+                </textarea>
```

Comment:
Fix formatting here, initial cursor placement is wrong
![image](https://user-images.githubusercontent.com/37161315/170653948-2486d5f4-2702-43d6-aa85-8b8f0a997739.png)
```suggestion
                <textarea 
                  name="Message"
                  placeholder="Message"
                  class="
                    contact-field
                    contact-field--last
                    form-field 
                    form-field--text"
               ></textarea>
```